jsapp/js/dataInterface.es6

```diff
@@ -239,6 +239,44 @@ export var dataInterface;
       });
     },
 
+    /*
+     * TODO: #2767 Dynamic data sharing, match with implemented API
+     */
+    attachToParent(assetUid, data) {
+      return $ajax({
+        url: `${ROOT_URL}/api/v2/assets/${assetUid}/paired-data/`,
+        method: 'POST',
+        data: data,
+        contentType: 'application/json'
+      });
+    },
+    detachParent(attachmentUrl) {
+      return $ajax({
+        url: attachmentUrl,
+        method: 'DELETE',
+      });
+    },
+    getAttachedParents(assetUid) {
+      return $ajax({
+        url: `${ROOT_URL}/api/v2/assets/${assetUid}/paired-data/`,
+        method: 'GET',
+      });
+    },
+    getSharingEnabledAssets() {
+      return $ajax({
+        url: `${ROOT_URL}/api/v2/assets/?q=data_sharing__enabled:True`,
```

Comment:
Just being curious - why is it `True` and not `true`?

Reply:
Slipped by when I copied the API from Oli ;)